docs/docs/typescript/introduction.md

```diff
@@ -132,3 +132,21 @@ import { CustomModal } from '@adminUI/CustomModal'
 1. **Improved code readability**, by abstracting complex directory hierarchies, and having meaningful names for your imports.
 1. **Code maintainability**, aliases allow you to decouple your code from the file structure and more easily move files around, as they are not tied to the longer path.
 1. **Reduce boilerplate**, no more `../../src/components/modules/admin/common/ui/` 😮‍💨
+
+When you start writing tests for components that contain alias paths, you will need to add the following to your Jest configuration in `jest.config.js`:
+
+```js
+const config = {
+  rootDir: '../',
+  preset: '@redwoodjs/testing/config/jest/web',
+  moduleNameMapper: {
+    '^@startRW/(.*)$': '<rootDir>/web/src/components/admin/$1',
+  },
+}
+
+module.exports = config
+```
+
+::info
+There are 3 `jest.config.js` files within a Redwood project. There's one inside the `web` directory, one inside th `api` directory, and one at the root of the project. Since the alias I created is used within the `web` directory, I added the `moduleNameMapper` to the `jest.config.js` file within the `web` directory.
```

Comment:
```suggestion
There are 3 `jest.config.js` files within a Redwood project. There's one inside the `web` directory, one inside the `api` directory, and one at the root of the project. Since the alias I created is used within the `web` directory, I added the `moduleNameMapper` to the `jest.config.js` file within the `web` directory.
```